packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -737,5 +737,17 @@ export const mutation = mutationType({
         return data
       },
     })
+
+    t.boolean('setRunAllSpecs', {
+      description: 'List of specs to run for the "Run All Specs" Feature',
+      args: {
+        runAllSpecs: nonNull(list(nonNull(stringArg({})))),
+      },
+      resolve: async (source, args, ctx) => {
```

Comment:
```suggestion
      resolve: (source, args, ctx) => {
```

Reply:
Done in [b0ba5c2](https://github.com/cypress-io/cypress/pull/24683/commits/b0ba5c2b0e86e79183a9715b58e6fa22c9aa703a)